smart-lungs/src/components/question-form.tsx

```diff
@@ -0,0 +1,312 @@
+"use client";
+
+import { DialogHeader, DialogTitle } from "./ui/dialog";
+import * as z from "zod";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "./ui/form";
+import { Input } from "./ui/input";
+import { Button } from "./ui/button";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "./ui/select";
+import { cn } from "@/lib/utils";
+
+const formSchema = z.object({
+  name: z.string().min(2, { message: "Name must be at least 2 characters." }),
+  location: z.string().min(2, { message: "Location must be at least 2 characters." }),
+  age: z
+    .number()
+    .int()
+    .min(16, { message: "You must be at least 16 years old." }),
+  gender: z.enum(["male", "female"]),
+  drinking: z.enum(["never", "rarely", "often", "daily"]),
+  troubleSwallowing: z.enum(["never", "rarely", "often", "daily"]),
+  coughing: z.enum(["never", "rarely", "often", "daily"]),
+  chestPainLevel: z.enum(["none", "mild", "moderate", "severe"]),
+  fatigue: z.enum(["never", "rarely", "often", "daily"]),
+  smoking: z.enum(["never", "rarely", "often", "daily"]),
+});
```

Comment:
Questions needs to be different tho, but they can stay for now

Reply:
As mentioned in the PR, I want to look which can be kept and which can be added that are not in the designs yet